assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/custom-metadata/components/table/table.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { createColumnHelper } from '@tanstack/react-table'
+import { type CustomMetadata, useGetAssetCustomMetadataByIdQuery } from '@Pimcore/modules/asset/asset-api-slice.gen'
+import { Button } from 'antd'
+import { Icon } from '@Pimcore/components/icon/icon'
+import { Grid } from '@Pimcore/components/grid/grid'
+import { AssetContext } from '@Pimcore/modules/asset/asset-provider'
+import { useStyle } from './table.styles'
+
+interface CustomMetadataWithActions extends CustomMetadata {
+  actions: React.ReactNode
+}
+
+export const CustomMetadataTable = (): React.JSX.Element => {
+  const { t } = useTranslation()
+  const { id } = useContext(AssetContext)
+  const { styles } = useStyle()
+  const { data, isLoading } = useGetAssetCustomMetadataByIdQuery({ id: id! })
+
+  const removeMetadata = (metadata: CustomMetadata): void => {
+    console.log('removeMetadata', metadata)
+  }
+
+  // const updateMetadata = (metadata: CustomMetadata): void => {
+  //  console.log('updateMetadata', metadata)
+  // }
+
+  function onUpdateCellData ({ rowIndex, columnId, value, rowData }): void {
+    console.log({ rowIndex, columnId, value, rowData })
+    // updateMetadata(value)
+  }
```

Comment:
I guess here is still the save missing?
Should we clean up a bit and mark it as todo?

Reply:
this is just the implementation to already have the UI - no functions at all so yeah there is a save missing ;) - will be something for the followup if that's OK :100: